blockchain/blockchain.go

```diff
@@ -500,6 +503,29 @@ func (bc *blockchain) GetActionsFromAddress(addrStr string) ([]hash.Hash256, err
 	return getActionsBySenderAddress(bc.dao.kvstore, hash.BytesToHash160(addr.Bytes()))
 }
 
+// GetActionsFromIndex returns actions from index
+func (bc *blockchain) GetActionHashFromIndex(index uint64) (hash.Hash256, error) {
+	hash := hash.ZeroHash256
```

Comment:
Move this implementation to blockdao, and call blockdao here?

Reply:
no problem